.circleci/config.yml

```diff
@@ -381,19 +387,16 @@ jobs:
     executor: linux
     steps:
       - checkout
-      - attach_workspace:
-          at: /tmp/workspace
       - run:
           name: Verify versions
           command: |
-            tar -xzvf /tmp/workspace/protostar-Linux.tar.gz ./dist/protostar/info/constants.json
-            built_version=$(grep dist/protostar/info/constants.json -e "PROTOSTAR_VERSION\":" | awk '{print $2}' | tr -d ',')
+            build_version=$(grep pyproject.toml -e "^version =" | awk '{print $3}')
```

Comment:
This is mostly out of curiosity, but why do we have to use extended regex here?

Reply:
extended is `-E`, `-e` is just regular pattern :)